packages/instant/src/components/retry_button.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+import { SecondaryButton } from './secondary_button';
+
+export interface RetryButtonProps {
+    onClick: () => void;
+}
+
+export const RetryButton: React.StatelessComponent<RetryButtonProps> = props => {
```

Comment:
Do you foresee adding functionality to this in the future? If not, seems a bit overkill to me (you could just inline the JSX).

Reply:
This came out of the design decision documented here: https://github.com/0xProject/0x-monorepo/pull/1170/files#r227176413   I do agree it feels a little bit like overkill though